public/js/user-actions.js

```diff
@@ -13,13 +16,56 @@ function UserActions(localDispatcher) {
 }
 
 
-UserActions.prototype.setCurrentUser = function(user) {
-  console.log('UserActions: setting current user', user);
-  this.dispatcher.dispatch({
-    actionType: 'set-user',
-    user: user,
-  });
-};
+UserActions.prototype = assign({}, {
+
+  signIn: function(accessToken, opt) {
+    opt = opt || {};
+    if (!opt.jquery) {
+      opt.jquery = $;
+    }
+
+    opt.jquery.ajax({
+      data: {
+        access_token: accessToken,
+      },
+      method: 'post',
+      url: '/api/auth/sign-in/',
+      context: this,
+    }).then(function(data) {
+
+      console.log('login succeeded');
+      this.setCurrentUser({
+        email: data.buyer_email,
+        is_logged_in: true,
+      });
+
+      console.log('setting CSRF token for subsequent requests:',
+                  data.csrf_token);
+      opt.jquery.ajaxSetup({
+        headers: {
+          'X-CSRFToken': data.csrf_token,
+        },
```

Comment:
Yeah, agreed on the header thing. We're going to ditch jquery soon for [fetch](https://github.com/mozilla/payments-ui/issues/84) but I didn't want to change too much in this patch. I like the idea of putting the token on ``UserStore``